packages/2d/src/components/Polygon.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  // PossibleSpacing,
```

Comment:
There's some leftover commented code here and there

Reply:
Thanks. Fixed now.